recommendation/recommended-tags.json

```diff
@@ -0,0 +1,11 @@
+{
+    "tekton.dev": [
+        "tekton"
+    ],
+    "operator.knative.dev": [
```

Comment:
This doesn't cover all use cases. Atleast in my minikube knative is not discovered correctly. Below what i have. Maybe we can think of using only `knative.dev` or if this syntax can create any problem to add something more like `serving.knative` or `sources` ...

![image](https://user-images.githubusercontent.com/49404737/134507612-0577852d-b3b4-4abb-869a-7b9a4e404e7f.png)

Reply:
hm, yes that can be a problem, as CRD's can differs, it is depends on how you install them(with operator or not)